src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/RescindRecordComponentTest.kt

```diff
@@ -148,15 +153,15 @@ class RescindRecordComponentTest : ComponentTestBase() {
     expectNoVirusesWillBeFound()
     val recall = authenticatedClient.bookRecall(bookRecallRequest)
 
-    val putRequest = RescindRecordController.RescindDecisionRequest(
+    val decisionRequest = RescindRecordController.RescindDecisionRequest(
       true,
       "Some details",
       LocalDate.now(),
       base64EncodedDocumentContents,
       approvalFileName
     )
 
-    authenticatedClient.decideRescind(recall.recallId, ::RescindRecordId.random(), putRequest, HttpStatus.NOT_FOUND)
+    decideRescindFailure(recall, ::RescindRecordId.random(), decisionRequest, NOT_FOUND)
```

Comment:
Since we're now getting the ErrorResponse class, we could now assert on the error message